tests/src/test/scala/org/virtuslab/tests/bazel/OpenProjectTestBazelWithCmdLine.scala

```diff
@@ -0,0 +1,20 @@
+package org.virtuslab.tests.bazel
+
+import org.virtuslab.ideprobe.RunningIntelliJFixture
+import org.virtuslab.ideprobe.protocol.ProjectRef
+import org.virtuslab.tests.OpenProjectTestFixture
+
+object OpenProjectTestBazelWithCmdLine
+  extends BazelTestSuite
+    with OpenProjectTestFixture {
+
+  override def openProject(): ProjectRef = ProjectRef.Default
```

Comment:
I think this is a good place to put `intelliJ.probe.await(WaitLogic.emptyNamedBackgroundTasks())`, this will trigger the important waiting before giving control to tests, and openProject() is called right after startup.

Reply:
Thanks, it worked!